packages/nextjs/components/HooksExample.tsx

```diff
@@ -122,6 +153,45 @@ const HooksExample: React.FC = () => {
             )}
           </div>
         </div>
+        {/* ETH Balance */}
+        <div className="rounded-[5px] bg-base-100 border border-gradient p-4 relative shadow">
+          <div className="trapeze"></div>
+          <h2 className="text-xl font-semibold mb-4">
+            Contract ETH Balance (useScaffoldEthBalance)
+          </h2>
+          {isEthBalanceLoading ? (
+            <p>Loading contract...</p>
+          ) : contract ? (
+            <p>{ethBalance} ETH</p>
+          ) : (
+            <p>No contract loaded</p>
+          )}
+        </div>
+
+        {/* STRK Balance */}
+        <div className="rounded-[5px] bg-base-100 border border-gradient p-4 relative shadow">
+          <div className="trapeze"></div>
+          <h2 className="text-xl font-semibold mb-4">
+            Contract STRK Balance (useScaffoldStrkBalance)
+          </h2>
+          {isStrkBalanceLoading ? (
+            <p>Loading contract...</p>
```

Comment:
Kindly use a skeleton to indicate loading for better UX

Reply:
I was looking for what are Skeletons, while looking I saw that I needed to import `react-loading-skeleton` library. I just wanted to know if this is what you meant or if it is something else.

Reply:
You can see the home page when you load without connecting a wallet. You would see something blinking. That is called a skeleton. 

The gray thing is called skeletons.

![image](https://github.com/user-attachments/assets/638800e6-e65f-4d83-8432-2dc514ddf1e6)